test/EndToEnd/tests/GetPackageTest.ps1

```diff
@@ -497,6 +497,7 @@ function Test-GetInstalledPackageWithFilterReturnsCorrectPackage
 
 function Test-GetPackageUpdatesAfterSwitchToSourceThatDoesNotContainInstalledPackageId
 {
+    [SkipTest('https://github.com/NuGet/Home/issues/10254')]
```

Comment:
Do we have tracking issue for re-enable skipped test in the future?

Reply:
Yes, the issue is in the description.https://github.com/NuGet/Home/issues/10254